content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

```diff
@@ -163,7 +163,7 @@ Note that the kubeadm CLI command never touches this drop-in file.
 
 This configuration file installed by the `kubeadm`
 [package](https://github.com/kubernetes/release/blob/cd53840/cmd/krel/templates/latest/kubeadm/10-kubeadm.conf) is written to
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
+`/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf` and is used by systemd.
```

Comment:
no, that's a bug in kubernetes/release

https://github.com/kubernetes/release/issues/3392

/close
as duplicate, not a docs bug.

Reply:
please comment on that ticket.